src/components/cylc/TaskFilter.vue

```diff
@@ -54,7 +58,10 @@ export default {
   },
 
   props: {
-    modelValue: Object // { id, states }
+    modelValue: {
+      type: Object,
+      default: () => ({})
```

Comment:
If this prop is required, you can set `required: true` rather than using a blank default.